UnitedIncome/components
# InterstitialStepScreen and InterstitialScreen Tweaks / Action Bar

### Description
> Please provide a brief description of your changes below.

* Implements the action bar to the interstitial screen. 
* Fixes the alignment so the interstitials properly center align to the screen. This is done with a combination of vh and absolute positioning as flex wasn't appropriate.
* Lots of visual tweaks to the screens, worked with Monique.
* Added the `InterstitialStepScreen` component which handles the step variant.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at the screens, verify their props work correctly.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

![Screen Shot 2020-04-24 at 1 49 38 PM](https://user-images.githubusercontent.com/10888441/80243527-6428b700-8635-11ea-872c-37cc2d791e31.png)
![Screen Shot 2020-04-24 at 1 49 42 PM](https://user-images.githubusercontent.com/10888441/80243532-65f27a80-8635-11ea-9794-120bbebed953.png)
![Screen Shot 2020-04-24 at 1 49 46 PM](https://user-images.githubusercontent.com/10888441/80243533-6723a780-8635-11ea-8e50-baa2652d4696.png)

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
